app/uk/gov/hmrc/entrydeclarationintervention/repositories/InterventionRepo.scala

```diff
@@ -125,4 +140,14 @@ class InterventionRepoImpl @Inject()(appConfig: AppConfig)(implicit mongo: React
       .sort(Json.obj("receivedDateTime" -> 1))
       .cursor[InterventionIds]()
       .collect[List](maxDocs = appConfig.listInterventionsLimit, err = Cursor.FailOnError[List[InterventionIds]]())
+
+  def getExpireAfterSeconds: Future[Option[Long]] =
```

Comment:
Do we need this at the moment?

Reply:
I included this to be able to test that the repo had set this correctly, although I might be testing internal mongo things (setting up the index correctly) which isn't reallly necessary to have test coverage for

Reply:
Maybe at least `getExpireAfterSeconds` should be private to the package - and the name might confuse if someone is looking for TTL related methods they might see this. If you keep it with this name then it might be an idea to add a comment to say why it is there. (But not sure it's needed as we don't test that other indexes have been created.)